code/style.css

```diff
@@ -1,4 +1,150 @@
-html {
-  background: #0025ff;
-  color: #fff;
+
+main, header, section, aside, footer {
+  margin: 0;
+  padding: 20px;
+  border: none;
+  color: dimgray;
+  text-align: center;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+  font-size: 24px;
+/*     border: 1px solid #000000; */
 }
+
+main {
+  background: #C8D2D8;
+
+}
+
+.header {
+  background: #708A9D;
+}
+
+.news-card {
+  display: flex;
+  flex-direction: row;
+  width: auto;
+  height: 310px;
+  margin-bottom: 20px;
+}
+
+.style {
```

Comment:
this classname is a bit unclear, could you change it to one thats a bit more descriptive

Reply:
It's the styling of the cards, so I changed the class to "cards".